app/views/rooms/cant_create_rooms.html.erb

```diff
@@ -17,10 +17,10 @@
 
 <div id="cant-create-room-wrapper" class="background h-100 cant-create-room-wrapper">
     <div class="container h-100">
-        <div class="row h-100 align-items-center">
+        <div class="row h-100 pt-7 align-items-center">
```

Comment:
The `pt-7` can be removed from here and instead added to the media query as `padding-top: 3rem`